interfaces/repo.go

```diff
@@ -862,11 +877,6 @@ func (e *BadInterfacesError) Error() string {
 // Unknown interfaces and plugs/slots that don't validate are not added.
 // Information about those failures are returned to the caller.
 func (r *Repository) AddSnap(snapInfo *snap.Info) error {
-	err := snap.Validate(snapInfo)
```

Comment:
If you choose to keep snap validation out of interface validation then please this code here.

Reply:
Done.